docs/2014/reporting-services/tutorial-introducing-expressions.md

```diff
@@ -13,697 +13,707 @@ ms.author: maghan
-  
-## Requirements  
- For information about requirements, see [Prerequisites for Tutorials &#40;Report Builder&#41;](../reporting-services/report-builder-tutorials.md).  
-  
-##  <a name="Setup"></a> 1. Create a Table Report and Dataset from the Table or Matrix Wizard  
- Create a table report, a data source, and a dataset. When you lay out the table, you will include only a few fields. After you complete the wizard you will manually add columns. The wizard makes it easy for you to lay out the table and apply a style.  
-  
-> [!NOTE]  
->  In this tutorial, the query contains the data values, so that it does not need an external data source. This makes the query quite long. In a business environment, a query would not contain the data. This is for learning purposes only.  
-  
-> [!NOTE]  
->  In this tutorial, the steps for the wizard are consolidated into one procedure. For step-by-step instructions about how to browse to a report server, choose a data source, and create a dataset, see the first tutorial in this series: [Tutorial: Creating a Basic Table Report &#40;Report Builder&#41;](../reporting-services/tutorial-creating-a-basic-table-report-report-builder.md).  
-  
-#### To create a new table report  
-  
-1.  Click **Start**, point to **Programs**, click [!INCLUDE[ssCurrentUI](../includes/sscurrentui-md.md)]**Report Builder**, and then click **Report Builder**.  
-  
-     The **Getting Started** dialog box appears.  
-  
-    > [!NOTE]  
-    >  If the **Getting Started** dialog box does not appear, from the **Report Builder** button, click **New**.  
-  
-    > [!NOTE]  
-    >  If you prefer using the ClickOnce version of Report Builder, open Report Manager and click **Report Builder**, or go to a SharePoint site on which Reporting Services content types such as reports are enabled and click **Report Builder Report** on the **New Document** menu on the **Documents** tab of a shared documents library.  
-  
-2.  In the left pane, verify that **New Report** is selected.  
-  
-3.  In the right pane, click **Table or Matrix Wizard**.  
-  
-4.  On the **Choose a dataset** page, click **Create a dataset**.  
-  
-5.  Click **Next**.  
-  
-6.  On the **Choose a connection to a data source** page, select a data source that is type **SQL Server**. Select a data source from the list or browse to the report server to select one.  
-  
-7.  Click **Next**.  
-  
-8.  On the **Design a query** page, click **Edit as Text**.  
-  
-9. Paste the following query into the query pane:  
-  
-    ```  
-    SELECT 'Lauren' AS FirstName,'Johnson' AS LastName, 'American Samoa' AS StateProvince, 1 AS CountryRegionID,'Unknown' AS Gender, CAST(9996.60 AS money) AS YTDPurchase, CAST('2010-6-10' AS date) AS LastPurchase  
-    UNION SELECT'Warren' AS FirstName, 'Pal' AS LastName, 'New South Wales' AS StateProvince, 2 AS CountryRegionID, 'Male' AS Gender, CAST(5747.25 AS money) AS YTDPurchase, CAST('2010-7-3' AS date) AS LastPurchase  
```

Comment:
Sorry, it might have gotten missed from the smartdiff view, but this was the original fix. The 'Warren' is stuck to the `SELECT`, so this code block isn't valid